front/src/modules/rollingStock/components/RollingStockCard/RollingStockCard.tsx

```diff
@@ -9,7 +9,7 @@ import { MdLocalGasStation } from 'react-icons/md';
 import { LazyLoadComponent } from 'react-lazy-load-image-component';
 
 import type {
-  RollingStockComfortType,
+  Comfort,
   LightRollingStockWithLiveries,
 } from 'common/api/osrdEditoastApi';
```

Comment:
To fix your build issue

```suggestion
import type { Comfort, LightRollingStockWithLiveries } from 'common/api/osrdEditoastApi';
```